python_src/src/lamp_py/runtime_utils/env_validation.py

```diff
@@ -0,0 +1,63 @@
+import os
+from typing import List, Optional
+
+from .process_logger import ProcessLogger
+
+
+def validate_environment(
```

Comment:
updated validate env function.

validate db checks for environment variables required for connecting to a database.
optional variables are logged if they exist.